Bumerang/app/src/main/AndroidManifest.xml

```diff
@@ -44,14 +44,16 @@
             android:windowSoftInputMode="adjustPan"/>
 
         <activity
+
```

Comment:
Nitpick, but just watch out for adding random spaces or newlines. It makes it harder to find the important changes. There's no need to bother changing it for now though, just for future reference.